packages/rocketchat-setup-wizard/client/setupWizard.js

```diff
@@ -74,6 +74,14 @@ const persistSettings = (state, callback) => {
 				_id: 'Statistics_reporting',
 				value: state.registerServer,
 			},
+			{
+				_id: 'Apps_Framework_enabled',
```

Comment:
this will disable apps after wizard completion if chose to not register the server, but there is nothing preventing the admin from enabling it afterwards even without registering the server.

Reply:
So, what is your recommendation?

Reply:
I think the only real option would be to disable apps setting to turn on unless registered... but I don't know if we actually want to do that.  Because I think we still want apps to be installable regardless.  

But they might not be able to talk to marketplace without registering?

Reply:
> But they might not be able to talk to marketplace without registering?

from what is said on the setup wizard, this is what I expect to happen. might be accomplished later on though, depending on the current expectations.